src/app/components/Canvas.tsx

```diff
@@ -255,6 +285,28 @@ export default function Canvas() {
           />
           <span>{Math.round((eyebrowsLevel * 100) / 13)} %</span>
         </div>
+        <div>
+          <Form.Label>回転</Form.Label>
+          <Form.Check
+            inline
+            type="switch"
+            onChange={handleRotationSwitch}
+            style={{ width: "20rem", marginLeft: "18px", marginRight: "10px" }}
+          />
+        </div>
+        <div>
+          <Form.Label>回転スピード</Form.Label>
+          <Form.Range
+            value={rotationSpeed}
+            onChange={handleRotationSpeedChange}
+            min={0.1}
+            max={10}
+            step={0.1}
+            default={1}
```

Comment:
The correct attribute seems to be `defaultValue`.
Thus, this line should be:
```typescript
defaultValue={1}
```

Reply:
Build log:
```
[20:07:55.298] Running build in Washington, D.C., USA (East) – iad1
[20:07:55.450] Cloning github.com/yude/gyutto-face (Branch: puhitaku/rotate, Commit: b45a7e7)
[20:07:55.961] Cloning completed: 507.003ms
[20:07:58.776] Restored build cache
[20:07:58.898] Running "vercel build"
[20:08:00.353] Vercel CLI 34.2.7
[20:08:00.779] Installing dependencies...
[20:08:01.239] yarn install v1.22.19
[20:08:01.317] warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[20:08:01.321] [1/4] Resolving packages...
[20:08:01.555] [2/4] Fetching packages...
[20:08:21.176] [3/4] Linking dependencies...
[20:08:21.178] warning " > bootstrap@5.3.3" has unmet peer dependency "@popperjs/core@^2.11.8".
[20:08:21.180] warning "react-konva > react-reconciler@0.29.2" has incorrect peer dependency "react@^18.3.1".
[20:08:26.850] [4/4] Building fresh packages...
[20:08:27.318] Done in 26.08s.
[20:08:27.363] Detected Next.js version: 13.4.4
[20:08:27.368] Running "yarn run build"
[20:08:27.561] yarn run v1.22.19
[20:08:27.598] $ next build
[20:08:28.170] - info Creating an optimized production build...
[20:08:37.210] (node:337) [DEP_WEBPACK_MODULE_UPDATE_HASH] DeprecationWarning: Module.updateHash: Use new ChunkGraph API
[20:08:37.211] (Use `node --trace-deprecation ...` to show where the warning was created)
[20:08:40.165] - info Compiled successfully
[20:08:40.168] - info Linting and checking validity of types...
[20:08:43.114] 
[20:08:43.115] ./src/app/components/Canvas.tsx
[20:08:43.115] 116:7  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
[20:08:43.115] 116:7  Warning: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  jsx-a11y/alt-text
[20:08:43.115] 
[20:08:43.115] info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
[20:08:45.672] Failed to compile.
[20:08:45.672] 
[20:08:45.672] ./src/app/components/Canvas.tsx:305:13
[20:08:45.673] Type error: Type '{ value: number; onChange: (e: ChangeEvent<HTMLInputElement>) => void; min: number; max: number; step: number; default: number; style: { width: string; marginLeft: string; marginRight: string; }; }' is not assignable to type 'IntrinsicAttributes & FormRangeProps & RefAttributes<HTMLInputElement>'.
[20:08:45.677]   Property 'default' does not exist on type 'IntrinsicAttributes & FormRangeProps & RefAttributes<HTMLInputElement>'.
[20:08:45.747] error Command failed with exit code 1.
[20:08:45.747] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[20:08:45.793] Error: Command "yarn run build" exited with 1
[20:08:47.079] 
```